packages/playground/cli/src/cli.ts

```diff
@@ -104,7 +104,7 @@ async function run() {
 		})
 		.option('debug', {
 			describe:
-				'Return PHP error log content if an error occurs while building the site.',
+				'Print PHP error log content if an error occurs during Playground boot.',
```

Comment:
Related discussion https://github.com/WordPress/wordpress-playground/pull/1983#discussion_r1829169712

Reply:
Thank you! a `--debug` flag that doesn't cover the entire lifecycle of the app seems confusing to me. What do you think about either renameing it to `--debug-boot` or making it work after the boot?

Reply:
I will add request debugging. Here is an issue to track it https://github.com/WordPress/wordpress-playground/issues/1988